src/Service.Tests/CosmosTests/MutationTests.cs

```diff
@@ -252,62 +255,79 @@ public async Task MutationMissingRequiredPartitionKeyValueReturnError()
         }
 
         /// <summary>
-        /// Mutation can be performed on the authorized fields because the
-        /// field `id` is an included field for the create operation on the anonymous role defined
-        /// for entity 'earth'
+        /// Create Mutation performed on the fields with different auth permissions
+        /// It throws permission denied error if role doesn't have permission to perform the operation
+        /// 'item-level-permission-role' is not applicable for create operation
```

Comment:
unclear why you need to include note on 'item-level-permission-role' here. Out of context, doesn't make sense to reference.

Reply:
I added that while testing item level auth..but you are right not related to this PR.